pkg/workload/workload.go

```diff
@@ -28,6 +28,7 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	apimeta "k8s.io/apimachinery/pkg/api/meta"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/api/equality"
```

Comment:
You can resolve CI errors with `make lint-fix` command in your local, I believe.

https://prow.k8s.io/view/gs/kubernetes-ci-logs/pr-logs/pull/kubernetes-sigs_kueue/5110/pull-kueue-verify-main/1915868931055161344